pkg/apis/policy/v1alpha1/types_certificaterequestpolicy.go

```diff
@@ -172,118 +174,123 @@ type CertificateRequestPolicyAllowedX509Subject struct {
+	// An omitted field implies no minimum constraint on duration.
 	// +optional
 	MinDuration *metav1.Duration `json:"minDuration,omitempty"`
 
 	// MaxDuration defines the maximum duration a certificate may be requested
 	// for.
-	// Values are inclusive (i.e. a max value of `1h` will accept a duration of
-	// `1h`). MaxDuration and MinDuration may be the same value.
-	// An omitted field or value of `nil` permits any maximum duration.
-	// If MaxDuration is defined, a duration _must_ be requested on the
-	// CertificateRequest.
+	// Values are inclusive (i.e. a value of `1h` will accept a duration of
+	// `1h`). MinDuration and MaxDuration may be the same value.
+	// If set, a duration _must_ be requested on the CertificateRequest.
+	// An omitted field implies no maximum constraint on duration.
 	// +optional
 	MaxDuration *metav1.Duration `json:"maxDuration,omitempty"`
 
-	// PrivateKey defines the shape of permissible private keys that may be used
-	// for the request with this policy.
-	// An omitted field or value of `nil` permits the use of any private key by
-	// the requestor.
+	// PrivateKey defines constraints on what shape of private key a
+	// CertificateRequest is allowed to use for its request.
+	// An omitted field implies no private key shape constraints.
 	// +optional
 	PrivateKey *CertificateRequestPolicyConstraintsPrivateKey `json:"privateKey,omitempty"`
 }
 
 // CertificateRequestPolicyConstraintsPrivateKey defines constraints on what
-// shape of private key is permissible for a CertificateRequest to have used
-// for its request.
+// shape of private key a CertificateRequest is allowed to use for its request.
 type CertificateRequestPolicyConstraintsPrivateKey struct {
 	// Algorithm defines the allowed crypto algorithm that is used by the
 	// requestor for their private key in their request.
-	// An omitted field or value of `nil` permits any Algorithm.
+	// An omitted field permits any Algorithm.
 	// +optional
 	Algorithm *cmapi.PrivateKeyAlgorithm `json:"algorithm,omitempty"`
 
-	// MinSize defines the minimum key size a requestor may use for their private
-	// key.
+	// MinSize defines the minimum key size a requestor may use for their
+	// private key.
 	// Values are inclusive (i.e. a min value of `2048` will accept a size
 	// of `2048`). MinSize and MaxSize may be the same value.
-	// An omitted field or value of `nil` permits any minimum size.
+	// An omitted field permits any minimum size.
 	// +optional
 	MinSize *int `json:"minSize,omitempty"`
 
-	// MaxSize defines the maximum key size a requestor may use for their private
-	// key.
+	// MaxSize defines the maximum key size a requestor may use for their
+	// private key.
 	// Values are inclusive (i.e. a min value of `2048` will accept a size
 	// of `2048`). MaxSize and MinSize may be the same value.
-	// An omitted field or value of `nil` permits any maximum size.
+	// An omitted field permits any maximum size.
```

Comment:
```suggestion
	// An omitted field implies no additional maximum constraint on size.
```

Reply:
> Again, not a fan of implies

😉

Reply:
The problem is that it isn't permitting any maximum size, it's permitting the size to be as large as the requester likes.

```suggestion
	// An omitted field allows size to be as large as possible.
```